src/main/java/com/google/sps/servlets/GrantLocationServlet.java

```diff
@@ -0,0 +1,24 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/grant-location")
+public class GrantLocationServlet extends HttpServlet {
+
+ /**
+  * Given the correlator, this will update the database to reflect that the user allowed their location to be accessed.
```

Comment:
(hard to tell on github you might be ok already)
Be sure to limit lines to <=100 characters (80 for javascript but 100 for java)

Reply:
i just set up an extension for that so it should be taken care of for the future!